dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    registries:
+      - gh-packages
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    allow:
+      - dependency-name: '@typeform*'
+
+registries:
+  gh-packages:
+    type: npm-registry
+    url: https://npm.pkg.github.com
+    token: ${{ secrets.GH_TOKEN }}
```

Comment:
Working with OPS in https://jira.typeform.tf/browse/INFRA-3372 to have this secret available in the org

Reply:
Done! Already available 😄